src/WebJobs.Script/Host/ScriptHost.cs

```diff
@@ -470,6 +470,11 @@ protected virtual void Initialize()
                 var functionMetadata = ReadFunctionMetadata(ScriptConfig, TraceWriter, _startupLogger, FunctionErrors, _settingsManager);
                 var usedBindingTypes = DiscoverBindingTypes(functionMetadata);
 
+                if (!ScriptConfig.LazyLoadExtensions)
```

Comment:
Well, nothing really comes of the exercising because there are no side effects from it. You're just throwing the result away, so I'm not sure why you'd compute it :)